test/functional/buildAndPackage/playlist.xml

```diff
@@ -24,6 +24,16 @@
 			<group>functional</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>Jre_Java_Version</testCaseName>
+		<command>$(TEST_JDK_HOME)/../j2re-image/bin/java -version; $(TEST_STATUS)</command>
```

Comment:
Will this be ok on macos which has a `Content/Home` at the top level of the downloaded tarball? Assuming it will be safe across all platforms I'm ok with it, but perhaps we should consider a `$(JRE_COMMAND)` definition in the future as an alternative